driver-core/src/test/unit/com/mongodb/internal/connection/MessageHelper.java

```diff
@@ -105,6 +114,10 @@ public static String getApiVersionField() {
         return getServerApi() == null ? "" : ", \"apiVersion\": \"" + getServerApi().getVersion().getValue() + "\"";
     }
 
+    public static String getDbField(final String databaseName) {
```

Comment:
Is this related to hello?

Reply:
Loosely.  It's related to the change in CommandMessage.getOpCode where we now use OP_MSG if API version is set.  If API version is non-null, we use OP_MSG, and when we use OP_MSG the command document includes a $db field.

Reply:
ACK - thanks for the clarification